keyboards/sharkoon/skiller_sgk50_s4/keyboard.json

```diff
@@ -137,21 +144,23 @@
             {"matrix": [0, 10], "x": 172, "y": 0, "flags": 4},
             {"matrix": [0, 11], "x": 190, "y": 0, "flags": 4},
             {"matrix": [0, 12], "x": 207, "y": 0, "flags": 4},
-            {"matrix": [0, 13], "x": 224, "y": 0, "flags": 4}
+            {"matrix": [0, 13], "x": 210, "y": 0, "flags": 4},
+            {"matrix": [0, 14], "x": 210, "y": 0, "flags": 4},
+            {"matrix": [0, 15], "x": 224, "y": 0, "flags": 4}
         ],
         "max_brightness": 110,
         "sleep": true,
         "val_steps": 28
     },
     "url": "",
     "usb": {
-        "device_version": "1.0.0",
+        "device_version": "1.1.0",
         "pid": "0x1020",
         "suspend_wakeup_delay": 1000,
         "vid": "0x6332"
     },
     "ws2812": {
-        "pin": "A8"
+        "pin": "B15"
```

Comment:
If this is a new hardware revision, it should be added as a new revision rather than config changes.

Reply:
The previous version was likely not released to users, so this modification falls under the same version.

Reply:
@drashna Can you continue the review?  Or do you need me to do something?